### PR TITLE
Fix pylint's too-few-public-methods

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: h/models/feature_cohort.py [hunk 1]
```diff
@@ -18,9 +18,6 @@ class FeatureCohort(Base, mixins.Timestamps):
         "Feature", secondary="featurecohort_feature", backref="cohorts"
     )
 
-    def __init__(self, name):
```
Comment (marcospri) [inline h/models/feature_cohort.py:21]: I don't think this is necessary in a model.

Comment (marcospri) [reply h/models/feature_cohort.py:21]: I also think this is third PR where I found it and removed it.

**Review by jon-betts — CHANGES_REQUESTED** [review]
We've discussed this before, and you might want to get @seanh's take as well, but I think we should globally disable this warning.

Honestly it's just part of our coding style that we don't care about this. We often use objects as a way to collect functionality together and expose a single method on something. Or any other number of times it's perfectly reasonable.

**Comment by marcospri** [comment]
> We've discussed this before, and you might want to get @seanh's take as well, but I think we should globally disable this warning.
> 
> Honestly it's just part of our coding style that we don't care about this. We often use objects as a way to collect functionality together and expose a single method on something. Or any other number of times it's perfectly reasonable.

I was thinking "this is bullshit" halfway in but was too invested to add the rule to pylintrc then! (jk, its pretty straightforward to add them)

File: .pylintrc [hunk 2]
```diff
@@ -16,6 +16,16 @@ disable=bad-continuation,
...
+        # We use isort to sort and group our imports, so we don't need PyLint to
+        # check them for us.
+        ungrouped-imports,
+
+        # We use Black to format our code automatically, so we don't need PyLint to
+        # check formatting for us.
+        bad-continuation,
+        line-too-long,
+
+        too-few-public-methods
```
Comment (marcospri) [inline .pylintrc:28]: This is the change that matches the original PR scope but added the rest of the config change here too.

**Review by jon-betts — APPROVED** [review]

**Pull request merged**